public/robots.txt

```diff
@@ -1,4 +1,7 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 User-Agent: *
-Allow: /exchange_rates
 Disallow: /
+Allow: /exchange_rates
+Allow: /tools
```

Comment:
are these 3 links a requirement? can you link the ticket? I think the requirement is to only allow /exchange_rates?
Allow: /tools
Allow: /news
Allow: /help

Reply:
I think it would be a good idea to get Google search console access or reregister the website on a new Google search console account before pushing to Robots.txt again

Reply:
No they are not, but I think we should include those pages:
my understanding is that we must not allow indexing of the goods, but it's OK  index the other pages like exchange_rates and the other pages that can be useful to the users.

For example, it makes sense that the user who searches for:
"Trade tariff news bulletin" to get as a result the page https://www.trade-tariff.service.gov.uk/news.